recipes/xgboost/meta.yaml

```diff
@@ -137,22 +137,6 @@ outputs:
       imports:
         - xgboost
 
-  - name: py-xgboost
-    build:
-      ignore_run_exports_from:
-        - nvcc_linux-64 # [linux64]
-        - nvcc_linux-aarch64 # [aarch64]
-    requirements:
-      host:
-        - python
-        - nccl >=2.5             # [xgboost_proc_type == "gpu"]
-      run:
-        - python
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
```

Comment:
This part in particular makes me wonder about this. Maybe it is leftover from [these]( https://github.com/conda-forge/xgboost-feedstock/blob/18e55d04c7dfd89e94b0c21c047632ab56889548/recipe/meta.yaml#L98-L119 )?

Reply:
certainly seems plausible!